lib/rubocop/rspec/align_let_brace.rb

```diff
@@ -34,7 +35,7 @@ def target_column_for(let)
       def let_group_for(let)
         adjacent_let_chunks.detect do |chunk|
           chunk.any? do |member|
-            member == let && member.loc.line == let.loc.line
+            member == let && same_line?(member, let)
```

Comment:
[`same_line?` is around for ages](https://github.com/rubocop/rubocop/commit/24e798efeb#diff-fd0332e5689d0ab642df39f706a915663f5d8764d4fa4442d36a291b16de70fbR265).